data/zilliqa/assets/zil19kjcp6d9v3ra0rec0u4p5a9gucgzlqqrvt2l40/meta.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "TRUSTMECOIN",
+  "symbol": "TMC",
+  "web": "https://trustmecoin.tech",
+  "explorer": "https://viewblock.io/zilliqa/address/zil19kjcp6d9v3ra0rec0u4p5a9gucgzlqqrvt2l40",
+  "links": {
+    "github": "https://github.com/Trustmecoin",
+    "medium": "trustmecoin.medium.com",
+    "twitter": "https://twitter.com/TrustmeCoin",
+    "telegram": "https://t.me/trustmeofficial",
+    
+  },
+  "description": "TrustMeCoin (TMC) is the Community Token decentralized trading protocol base on ZilliqaChain

```

Comment:
descriptions should be short

Reply:
how many word sir?
i change and send PR again sir??

Reply:
Just update this PR, don't create a new one